views/js/controller/creator/creator.js

```diff
@@ -24,6 +24,7 @@ define([
     'lodash',
     'helpers',
     'i18n',
+    'services/features',
```

Comment:
chore: I think we need to update the dependency to tao-core version > 50.10.0 where the service was added : https://github.com/oat-sa/tao-core/releases/tag/v50.10.0  in the composer.json of the extension
And I think we forgot to do it in taoQtiItem too :disappointed:

Reply:
Updated the dependency here and in taoQtiItem as well, please check this PR https://github.com/oat-sa/extension-tao-itemqti/pull/2119